### ISSUE TITLE
[BUG] : run dbt commands

### ISSUE THREAD

**Issue by philiphagglund** [datnguye/dbterd#106]
Hi,

Previously it worked to use `dbterd run --dbt -s tag:<tag>` but now i get the following message:

``` 
$ dbterd run --dbt -s tag:hub
2024-05-02 11:07:00,093 - dbterd - INFO - Run with dbterd==1.13.2 (main.py:54)
2024-05-02 11:07:00,101 - dbterd - ERROR - Unsupported Selection found: tag (base.py:100)
Usage: dbterd run [OPTIONS]
Try 'dbterd run -h' for help.

Error: Unsupported Selection found: tag
``` 

When testing the generated output from dbterd it works by adding the `tag:<tag>` into the string:

failed:
``` 
dbterd - DEBUG - Invoking: dbt --quiet --log-level none ls --resource-type model --select <model> --project-dir
``` 
working:
```
dbt --quiet --log-level none ls --resource-type model --select tag:<tag> --project-dir
```

**To Reproduce**

1. tag a model
2. run `bterd run --dbt -s tag:<tag>`

**Expected behavior**
What all objects within specific tag will be be included in dbterd

**Comment by datnguye** [comment]
Thanks @philiphagglund for this! 

I will check it out shortly assuming you were trying with the latest version v1.13, let me know otherwise. Thanks

**Comment by philiphagglund** [comment]
> Thanks @philiphagglund for this!
> 
> I will check it out shortly assuming you were trying with the latest version v1.13, let me know otherwise. Thanks
```
$ dbterd --version
dbterd, version 1.13.2
```

**Comment by datnguye** [comment]
Hi @philiphagglund Sorry for this issue! I have fixed it with adding 1 more test case to prevent it.

It should be released soon in 1.13.3! 🙌

**Comment by datnguye** [comment]
Just published now v1.13.3

Please help to check it out and re-open the issue if the issue still persisted. Thanks!
